pages/common/vim.md

```diff
@@ -1,20 +1,28 @@
 # vim
 
-> Vi IMproved, a programmer's text editor, provides several modes for different kinds of text manipulation.
+> Vim (Vi IMproved), a programmer's command-line text editor, provides several modes for different kinds of text manipulation.
 > Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
 > More information: <https://www.vim.org>.
 
 - Open a file:
 
-`vim {{file}}`
+`vim {{path/to/file}}`
 
-- Enter text editing mode (insert mode):
+- Save a file:
 
-`<Esc>i`
+`:write`
```

Comment:
Would it be better to omit the `<Esc>` key like this command, for all commands in this file? 
I feel like they should all include it or not include the `<Esc>` at all.

I feel like putting the `<Esc>`key in those definitions assumes that the user is currently in a mode other than `normal`. I feel like only the actual `:commands` should be there.

What do you think?

Reply:
I agree with you.  I'll make the change in my PR.